R/AbstractGraphReporter.R

```diff
@@ -108,7 +108,7 @@ AbstractGraphReporter <- R6::R6Class(
     private = list(
         plotNodeColorScheme = list(
             field = NULL
-            , pallete = '#97C2FC'
+            , palette = '#97C2FC'
```

Comment:
😱 which of these happened:

1. This was always broken and you fixed it
2. This just broke because some function we call changed its API
3. We were always technically wrong but it was fine because of R's partial matching thing
4. This wasn't broken and isn't broken but you're just cleaning up

Reply:
4. This wasn't broken and isn't broken but I'm just cleaning it up.

The use of this parameter/variable is entirely internal to pkgnet.